meta/main.yml

```diff
@@ -2,22 +2,28 @@
 
 dependencies:
 
+  - role: debops.apt
+    apt_repositories:
+    - repo: 'deb {{owncloud_apt_repo_base}}/{{owncloud_edition}}:/{{owncloud_release}}/{{ansible_distribution}}_{{ansible_distribution_major_version}}.0/ /'
+      state: present
+    apt_keys:
+    - url: '{{owncloud_apt_repo_base}}/{{owncloud_edition}}:/{{owncloud_release}}/{{ansible_distribution}}_{{ansible_distribution_major_version}}.0/Release.key'
+      state: present
```

Comment:
You don't really need to use `debops.apt` role this way. Since the repository is required anyway, using `apt_key` and `apt_repository` Ansible modules is cleaner and uses less time than re-running the whole APT role.

Have you checked if `debops.owncloud` role works with packages from Debian Jessie/Wheezy Backports?